### PR TITLE
{bio}[foss/2022a] CASPR v20200730 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by deniskristak**
adding CASPR easyconfig + deps
https://github.com/vscentrum/vsc-software-stack/issues/116

### CODE REVIEW

**Comment by deniskristak** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 9 out of 9 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/efa99a27f706cf70502e335a191dd1f1 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-GCC-11.3.0.eb [hunk 1]
```diff
@@ -24,11 +26,6 @@ toolchainopts = {'pic': True}
 
 default_easyblock = 'ConfigureMake'
 
-dependencies = [
-    ('GDGraph', '1.56'),
-    ('Perl', '5.34.1'),
```
Comment (boegel) [inline easybuild/easyconfigs/f/FASTX-Toolkit/FASTX-Toolkit-0.0.14-GCC-11.3.0.eb:29]: @deniskristak It looks like you're accidentally changing an existing easyconfig here?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/v/vispr/vispr-0.4.14-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+homepage = 'https://pypi.org/project/vispr/'
+
+description = """
+VISPR - A visualization framework for CRISPR data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = ['%(name)s-%(version)s.tar.gz']
+patches = ['vispr_fix_pandas_version_change.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/v/vispr/vispr-0.4.14-foss-2022a.eb:17]: @deniskristak Please rename the patch file to also include the `vispr` version to which it applies: `vispr-0.4.14_fix_pandas_version_change.patch`

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @deniskristak!

**Pull request merged**